scripts/find-tools

```diff
@@ -15,6 +15,7 @@ tags = {	'configure': "This script is used for the build configuration",
 		'env': "This script does some env stuff",
 		'mount': "This script mounts things",
 		'reformat': "This script reformats things",
+		'benchmark': "This script runs benchmarks",
```

Comment:
I wanted to add a proper header in my script, so I need a new category since it does not fit in any other listed here.